sanitize/schemas.go

```diff
@@ -61,7 +62,7 @@ func (s *Sanitizer) fetchEntitySchema(entityName string, field reflect.Value) (s
 		return entityName, kong.Schema{}, fmt.Errorf("kong client is not initialized")
 	}
 
-	if entityName == Plugin {
+	if entityName == Plugin || entityName == ConsumerGroupPlugin {
```

Comment:
Found consumer-group-plugin handling missing after adding the integration test for this specific case. Hence, added the minor corrections in this PR itself.